homeassistant/components/openweathermap/sensor.py

```diff
@@ -315,7 +316,9 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the state of the device."""
-        return self._weather_coordinator.data.get(self.entity_description.key, None)
+        return self._weather_coordinator.data[ATTR_API_CURRENT].get(
+            self.entity_description.key, None
+        )
```

Comment:
```suggestion
        return self._weather_coordinator.data[ATTR_API_CURRENT].get(self.entity_description.key)
```

`None` is the default

`ruff` may complain about the formatting of this suggestion

Reply:
done